uPortal-webapp/src/main/resources/properties/contexts/securityContext.xml

```diff
@@ -169,4 +169,19 @@
         </property>
     </bean>
 
+    <!--
+     | Spring-managed instance of CAS AuthenticationFilter. As above, this allows us to configure
+     | CAS authentication in the preferred approach. This is only needed for 'Guest Redirect to CAS'.
+     +-->
+    <bean name="casAuthenticationFilter" class="org.apereo.portal.spring.web.BypassFilterDecorator">
```

Comment:
Did the name change to `OptionalFilterDecorator`?

I'm getting...

```
Caused by: java.lang.ClassNotFoundException: org.apereo.portal.spring.web.BypassFilterDecorator
	at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1291)
	at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1119)
	at org.springframework.util.ClassUtils.forName(ClassUtils.java:250)
	at org.springframework.beans.factory.support.AbstractBeanDefinition.resolveBeanClass(AbstractBeanDefinition.java:401)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doResolveBeanClass(AbstractBeanFactory.java:1432)
	at org.springframework.beans.factory.support.AbstractBeanFactory.resolveBeanClass(AbstractBeanFactory.java:1377)
	... 43 common frames omitted
```

Reply:
Yep ... fixed.